src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -177,36 +203,54 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
 
   handleFilter = (breadcrumbFilterGroups: BreadcrumbFilterGroups) => () => {
     //types
-    const breadcrumbFilterGroupTypes = breadcrumbFilterGroups.filter(
-      breadcrumbFilterGroup => breadcrumbFilterGroup.groupType === 'type'
-    );
-
-    // TODO(Priscila): implement levels
-    // const breadcrumbFilterGroupLevels = breadcrumbFilterGroups
-    //   .filter(breadcrumbFilterGroup => breadcrumbFilterGroup.groupType === 'level')
-    //   .map(breadcrumbFilterGroup => breadcrumbFilterGroup.type);
+    const breadcrumbFilterGroupTypes = breadcrumbFilterGroups
+      .filter(
+        breadcrumbFilterGroup =>
```

Comment:
I kind of feel like some of these variable names _could_ be shorter. Feels a little like I'm reading C# haha

Reply:
yeah I got it...other colleagues said the same... I am going to rename them in a follow-up PR 😉